examples/demo.js

```diff
@@ -1,39 +1,48 @@
 import React, { Component } from 'react'
 import clsx from 'clsx'
 import Villain from '@/index.js'
-import {version} from '../package.json';
+import { version } from '../package.json'
 import './demo.css'
+import ToggleThemeButton from './ToggleThemeButton'
 
 class Demo extends Component {
-
   state = {
     file: '/build/testFile/Example-archive.zip',
     options: {
       preview: 0,
-      theme: {type: 'select', value: 'Light', options: ['Dark', 'Light']},
+      theme: { type: 'select', value: 'Light', options: ['Dark', 'Light'] },
       mangaMode: false,
       allowFullScreen: true,
       autoHideControls: false,
       allowGlobalShortcuts: false,
-    }
+    },
+    theme: 'light',
   }
 
   constructor(props) {
     super(props)
   }
 
+  toggleTheme = () => {
+    const theme = this.state.theme === 'light' ? 'dark' : 'light'
+    document.documentElement.classList.add('color-theme-in-transition')
+    this.setState({ theme })
```

Comment:
Use `options.theme` instead, there should be only one selector for changing the theme for component and example page

Reply:
okay, will make the changes